milestone-02/express-stripe/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const stripe = require('stripe')("");
+
+stripe.charges.create({
```

Comment:
Right now all that is happening is that a charge is submitted when the application boots if I am not mistaken.